src/components/Input/Input.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
```

Comment:
Here you don't need to import React, since you already have it in the index.js

Reply:
Good point! ;-) Thanks a lot.